src/rpcgen/trn_rpc_protocol.x

```diff
@@ -380,6 +380,8 @@ program RPC_TRANSIT_REMOTE_PROTOCOL {
 
                 int UPDATE_TX_STATS(rpc_trn_tx_stats_t) = 46;
                 rpc_trn_tx_stats_t GET_TX_STATS(rpc_trn_tx_stats_key_t) = 47;
+                int LOAD_TRANSIT_XDP_OFFLOAD(rpc_trn_xdp_intf_t) = 48;
```

Comment:
@phudtran IIRC did we not already have support in existing functions for loading / unloading in offload mode?